io.github.DamnedAngel.msx-tile-forge.yaml

```diff
@@ -0,0 +1,265 @@
+app-id: io.github.DamnedAngel.msx-tile-forge
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+
+command: msxtileforge
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-documents
+
+modules:
+  # tkinter dependencies
+  - name: tkinter
+    buildsystem: simple
+    build-options:
+      env:
+        CFLAGS: "-I/app/include/python3.12 -I/app/include/python3.12/internal -I/app/include"
+    build-commands:
+      - pip3 install --prefix=${FLATPAK_DEST} . --use-pep517 --no-build-isolation
+    sources:
+      - type: git
+        url: https://github.com/iwalton3/tkinter-standalone
+        commit: 35bde961500fc39ebc5587dcc032a6de31adfe4a
+    modules:
+      - name: python-internal-headers
+        buildsystem: simple
+        build-commands:
+          - mkdir -p /app/include/python3.12
+          - cp -r Include/* /app/include/python3.12/
```

Comment:
if it is getting clean-ed up anyway maybe install to `/app/include/python3` instead.